### PR TITLE
Feature/#5 split folder structure

### LINKED ISSUE #5: Library should be using correct folder structure
lib/dart_cose (exporting public files)
lib/src/* (contains all the files for this library)

### PR DESCRIPTION
**Pull request by vanlooverenkoen**
Fixed #5

### CODE REVIEW

File: lib/src/cose.dart [hunk 1]
```diff
@@ -239,7 +180,7 @@ class Cose {
             npk,
             Uint8List.view(signers.buffer, 0, signers.length),
             sigStructureBytes.buffer.asUint8List());
-        print(verified);
+        CoseLogger.print(verified);
       } else if (-35 == a) {
```
Comment (ikbendewilliam) [inline lib/src/cose.dart:184]: Give these values a name -35 -> sha384, -36 -> sha512
Also a better variable name than `a` would be nice

File: lib/src/model/cose_error_code.dart [hunk 2]
```diff
@@ -0,0 +1,12 @@
+enum CoseErrorCode {
+  none,
```
Comment (ikbendewilliam) [inline lib/src/model/cose_error_code.dart:2]: Capitalize enum values

Comment (vanlooverenkoen) [reply lib/src/model/cose_error_code.dart:2]: That would break the current release on pub.dev

File: lib/src/cose.dart (hunk 1 above)
Comment (vanlooverenkoen) [reply lib/src/cose.dart:184]: Yes indeed. I would like to update that as well. For now this pull requests splits up the library in multiple files & added an extra method `extractKid` from a pem string. But this should be updated in the future. As well as updating the analysis_options.yaml

**Review by ikbendewilliam — APPROVED** [review]

**Pull request merged**